configs/etc/kubehound-reference.yaml

```diff
@@ -29,11 +29,17 @@ collector:
   # file:
   #   # Directory holding the K8s json data files
   #   directory: /path/to/directory
+  #
+  #   # Target cluster name
+  #   cluster: <cluster name>
 
 #
 # General storage configuration
 #
 storage:
```

Comment:
Don't you want to also update the other config files as well? Just to simplify end user life :)

Reply:
the other configs don't use the file collector so this field is not required